source/backend/api/Areas/Reports/Models/Acquisition/CompensationFinancialReportModel.cs

```diff
@@ -91,9 +94,10 @@ public CompensationFinancialReportModel(PimsCompReqFinancial financial, Compensa
         {
             ExportBy = user.GetDisplayName();
             ExportDate = DateTime.Now.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture);
-            MinistryProject = GetMinistryProjectName(financial.CompensationRequisition?.AcquisitionFile?.Project);
-            Product = GetMinistryProductName(financial.CompensationRequisition?.AcquisitionFile?.Product);
-            AcquisitionNumberAndName = GetAcquisitionFileName(financial.CompensationRequisition?.AcquisitionFile);
+            MinistryProject = GetMinistryProjectName(financial.CompensationRequisition?.AcquisitionFile?.Project ?? financial.CompensationRequisition?.Lease?.Project);
+            Product = GetMinistryProductName(financial.CompensationRequisition?.AcquisitionFile?.Product ?? financial.CompensationRequisition?.Lease?.Product);
```

Comment:
This assignment may not work all the time, I would rather check for the LeaseId property or the AcquisitionFileId property to determine which file type the compensation requisition is for.